src/igl/opengl/IContext.cpp

```diff
@@ -1541,6 +1541,23 @@ void IContext::drawElements(GLenum mode, GLsizei count, GLenum type, const GLvoi
   APILOG_DEC_DRAW_COUNT();
 }
 
+void IContext::drawElementsInstanced(GLenum mode, GLsizei count, GLenum type, const GLvoid* indices, GLsizei instancecount) {
+    drawCallCount_++;
+
+    IGL_PROFILER_ZONE_GPU_COLOR_OGL("drawElementsInstanced()", IGL_PROFILER_COLOR_DRAW);
+
+    GLCALL(DrawElementsInstanced)(mode, count, type, indices, instancecount);
```

Comment:
This is not available on GLES 2 and GLES 3.0.

Reply:
This is available on GLES 3.0, not available on GLES 2. I will improve it.

Reply:
Sorry, not available in desktop OpenGL 3. Not GLES 3. :-)

https://registry.khronos.org/OpenGL-Refpages/gl4/html/glDrawElementsInstanced.xhtml